assets/js/base/components/cart-checkout/form-step/style.scss

```diff
@@ -12,6 +12,16 @@
 	.is-large & {
 		padding-right: $gap-large;
 	}
+
+	.wc-block-components-checkout-step__heading::after {
```

Comment:
I wasn't sure what this was for but I see it's for the border. This part doesn't work in Safari—expected?

Reply:
This creates a short grey line at the bottom of the `step__heading` - it _is_ present in Safari, but because `gap` is not supported it does not line up well, so not immediately obvious that it is there. I think we opt to leave this in, because when Safari supports `gap` it'll line up automatically and have the grey border next to the "Already have an account" text as it does in Chrome and Firefox.